deployment_script.py

```diff
@@ -1,10 +1,11 @@
 from pymongo import MongoClient
 import os
 from scripts._0_0_8_add_textbased_alignment import add_textbased_alignments
+from scripts._0_0_9_add_scenario_names import add_scenario_names
```

Comment:
I had to remove the import of `add_textbased_alignments` due to the character encoding issue we encountered when trying to get my dashboard running.  The import is no longer needed here anyway.   I assume it's a separate ticket to take the actions we discovered when trying to get my dashboard working.